tests/e2e/specs/admin-tracking.test.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * WordPress dependencies
+ */
+import { activatePlugin, deactivatePlugin, visitAdminPage } from '@wordpress/e2e-test-utils';
+import { URL } from 'url';
+import { resetSiteKit } from '../utils';
+
+describe( 'Providing client configuration', () => {
+
+	beforeAll( async() => {
+		await activatePlugin( 'google-site-kit' );
```

Comment:
The Site Kit plugin should actually be already active during E2E tests (done in the Docker setup). The only exception where it needs to be (de)activated is in the test for the plugin activation itself.

Reply:
That makes perfect sense, no longer activating/deactivating the plugin.